app.js

```diff
@@ -1,14 +1,34 @@
 
 document.getElementById('submit').addEventListener('click', function() {
-let name = document.getElementById("name").value;
-let dob = document.getElementById("dob").value;
-let email = document.getElementById("email").value;
-let phone = document.getElementById("phone").value;
-let city = document.getElementById("city").value;
-let institute = document.getElementById("institute").value;
+let name = document.getElementById("name");
+let dob = document.getElementById("dob");
+let email = document.getElementById("email");
+let phone = document.getElementById("phone");
+let city = document.getElementById("city");
+let institute = document.getElementById("institute");
 var gen = document.getElementsByName("gender");
 var qual = document.getElementsByName("c");
 
+
+    if (name.value == 0) {
+      window.alert("please enter your name");
+      return false;
+    }
+    else if(dob.value == 0) {
+        window.alert("please enter your date of birth");
+        return false;
+      }
+    else if(phone.value == 0) {
+        window.alert("please enter your contact number");
+        return false;
+      }
+    else if(email.value == 0) {

```

Comment:
email validation has to be done.

Form email input default provides these validations. use similar controls to handle validations out of the box